### PR TITLE
Test against newer php versions

### PR DESCRIPTION
**Pull request by BackEndTea**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,12 @@
 language: php
 
-sudo: false
-
-# whitelist for building branches
-branches:
-  only:
-    - master
-    - stable
-
```
Comment (daanstokhof) [inline .travis.yml]: Could you revert these changes? Or can you explain why you've chosen to remove these lines?

Comment (BackEndTea) [reply .travis.yml]: These lines cause travis to only. trigger on master or on stable branches, meaning merge requests wont be tested by travis automatically.
